Changelog.md

```diff
@@ -1,4 +1,18 @@
-## Ignition CMake 2.x
+## Gazebo CMake 2.x
+
+### Gazebo CMake 2.17.0 (2023-05-19)
+
+1. Use CONFIG in gz_add_benchmark to avoid Windows collisions
+    * [Pull request #341](https://github.com/gazebosim/ign-cmake/pull/341)
+
+1. LICENSE: add Apache 2.0 license text
+    * [Pull request #338](https://github.com/gazebosim/ign-cmake/pull/338)
+
+1. Disable protobuf warnings on protobuf target (#335)
+    * [Pull request #335) (#336](https://github.com/gazebosim/ign-cmake/pull/335) (#336)
```

Comment:
formatting

Reply:
maybe just this?

```
* [Pull request #335](https://github.com/gazebosim/ign-cmake/pull/335)
```

Reply:
ouch thanks! [447cb3d](https://github.com/gazebosim/gz-cmake/pull/351/commits/447cb3dc61b9cb12e0cfbce6a5fb14b0206f001b)